doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-test-scenario-export-import-spreadsheet-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='test-designer-test-scenario-export-import-spreadsheet-con']
+= Exporting and Importing a test scenario spreadsheet
```

Comment:
The title sounds more like a procedure rather than a concept so maybe pluralizing this? Importing should be lowercase because we use sentence case for titles.
s/Exporting and Importing a test scenario spreadsheet/Exporting and importing test scenario spreadsheets

Reply:
Alternative title idea:
"Test scenario spreadsheet data management"

Reply:
Ack. I will go with the first one. :)

Reply:
Done.